src/app/shared/modules/create-product-modal/create-product-modal.component.html

```diff
@@ -31,6 +31,46 @@ <h1 mat-dialog-title>Create new product</h1>
         {{ getErrorMessage("description") }}
       </mat-error>
     </mat-form-field>
+
+    <div formArrayName="counts">
+      <div
+        class="form-array"
+        *ngFor="let count of counts().controls; let i = index"
+        [formGroupName]="i"
+      >
+        <div>
+          <mat-form-field appearance="fill">
+            <mat-label>Location</mat-label>
+            <input matInput type="text" formControlName="location" />
```

Comment:
Don't leave a ` ` (space) before closing tag here and everywhere else.
`location" />` -> `location"/>`

Reply:
I checked all formatting rules, but I can't find where I can change or set it, also I checked some forums etc. but I didn't find the solution